packages/react-ui-components/src/stories/Components/SchemaForm.stories.tsx

```diff
@@ -546,3 +547,215 @@ CustomSelectWidgetConfigs.args = {
 		}
 	}
 };
+
+export const InlineForm = FormTemplate.bind(this);
+InlineForm.args = {
+	showErrorList: false,
+	liveValidate: true,
+	disabled: false,
+	allowDiscardChanges: true,
+	formData: {
+		team_info: [{}],
+		assets_oee: [{ asset_name: 'Demo Centrifugal Pump 10' }, { asset_name: 'kelvin simple asset' }]
+	},
+	schema: {
+		type: 'object',
+		properties: {
+			shift_info: {
+				title: 'Shift info',
+				type: 'object',
+				properties: {
+					name: {
+						title: 'Shift name',
+						type: 'string'
+					}
+					// start_at: {
```

Comment:
can we remove the commented out props?

Reply:
@ricardogoncalves89 Do you find a way to deal with datetime picker?